src/Movie-Card/Movie-Card.scss

```diff
@@ -0,0 +1,77 @@
+@import '../styles.scss';
+
+.movie-card {
+    height: 90%;
+    width: auto;
+    padding: 0;
+    margin: 20px;
+    border-radius: 5px;
+    position: relative;
+    padding-left: 10px;
+    align-content: center;
+    font-size: 0.8em;
+    font-family: sans-serif;
+    color: whitesmoke;
+    font-kerning: normal;
+    text-shadow: 0ch;
+    outline-color: black;
+         &:hover {
```

Comment:
Not sure where else to put the hover effect. It's not quite as consistent as I would like but I couldn't figure out a better place. Let me know if you have an idea!

Reply:
I think it needs to be on all the elements that have a hover effect and have the properties that affect that element.